specification/mobilenetwork/resource-manager/readme.python.md

```diff
@@ -3,16 +3,19 @@
 These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-``` yaml $(python)
-azure-arm: true
-license-header: MICROSOFT_MIT_NO_VERSION
-package-name: azure-mgmt-mobilenetwork
-namespace: azure.mgmt.mobilenetwork
-package-version: 1.0.0b1
-clear-output-folder: true
+```yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: Microsoft.MobileNetwork
+  package-name: mobilenetwork
+  package-version: 2022-11-01
+  clear-output-folder: true
 ```
 
-``` yaml $(python)
-no-namespace-folders: true
-output-folder: $(python-sdks-folder)/mobilenetwork/azure-mgmt-mobilenetwork/azure/mgmt/mobilenetwork
+```yaml $(python)
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt/mobilenetwork
```

Comment:
Please revert the change for readme.python.md, otherwise python pipeline will fail

Reply:
Done. @sathiyan-sivathas  do you know how the changes to readme.python.md were made? Are they auto-gen'ed?

Reply:
They were auto generated. The changes are in https://github.com/Azure/azure-rest-api-specs-pr/pull/8733/commits/35c2a06250528dd7e98b6c6dc8deb6755532a8e0#diff-ed40669cd89db725964018ebbbc8238aee14e6012559f266ee5b1c8e1dd27e9e.

The only change in the private PR is to change `package-version` to `2022-11-01`.

I think what's happened is that the python readme in the private repo is different to the one in the public repo and the tool that applies the private PR to the public repo has handled that badly.

Should we just update the private repo to match the public on @msyyc?